homeassistant/components/hyperion/const.py

```diff
@@ -28,3 +28,6 @@
 TYPE_HYPERION_CAMERA = "hyperion_camera"
 TYPE_HYPERION_LIGHT = "hyperion_light"
 TYPE_HYPERION_COMPONENT_SWITCH_BASE = "hyperion_component_switch"
+TYPE_HYPERION_COMPONENT_SENSOR_BASE = "hyperion_state"
```

Comment:
The name doesn't feel right, perhaps would `hyperion_visible_priority` be better? The constant name should be more specific, it's not really a "base" unless you implement the idea below about making a generic base sensor class.